common/app/routes/Challenges/Help-Modal.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button, Modal } from 'react-bootstrap';
+
+import ns from './ns.json';
+import {
+  createQuestion,
+  openHelpChatRoom,
+  closeHelpModal,
+  helpModalSelector
+} from './redux';
+
+const mapStateToProps = state => ({ isOpen: helpModalSelector(state) });
+const mapDispatchToProps = { createQuestion, openHelpChatRoom, closeHelpModal };
+const methodologyUrl = 'https://forum.freecodecamp.org/t/the-read-search-ask-methodology-for-getting-unstuck/137307'; // eslint-disable-line max-len
+
+const propTypes = {
+  closeHelpModal: PropTypes.func,
+  createQuestion: PropTypes.func,
+  isOpen: PropTypes.bool,
+  openHelpChatRoom: PropTypes.func
+};
+
+export class HelpModal extends PureComponent {
+  render() {
+    const {
+      isOpen,
+      closeHelpModal,
+      openHelpChatRoom,
+      createQuestion
+    } = this.props;
+    return (
+      <Modal
+        show={ isOpen }
+        >
+        <Modal.Header className={ `${ns}-list-header` }>
+          Ask for help?
+          <span
+            className='close closing-x'
+            onClick={ closeHelpModal }
+            >
+            ×
+          </span>
+        </Modal.Header>
+        <Modal.Body className='text-center'>
+          <h3>
+          If you've already tried the&nbsp;Read-Search-Ask&nbsp;method,
+          then you can ask for help on the freeCodeCamp forum.
+          </h3>
+          <Button
+            block={ true }
+            bsSize='lg'
+            bsStyle='primary'
+            href={ methodologyUrl }
+            target='_blank'
+            >
+            Learn about the Read-Search-Ask Methodology
```

Comment:
Clicking this button should close the modal

Reply:
Should clicking "Read "How to Report a Bug"" close bug modal as well?

Reply:
Possibly, but that should be separate to this PR :+1: